Source/Core/Core/ActionReplay.cpp

```diff
@@ -314,8 +314,6 @@ static void VLogInfo(std::string_view format, fmt::format_args args)
     return;
 
   const bool use_internal_log = s_use_internal_log.load(std::memory_order_relaxed);
-  if (Common::Log::MAX_LOGLEVEL < Common::Log::LogLevel::LINFO && !use_internal_log)
-    return;
```

Comment:
This one isn't unreachable, `MAX_LOGLEVEL` is redefined based on either `_DEBUG` or `DEBUGFAST`, where it may go below `LINFO`.

Reply:
I think the reason is because `is_internal_log` is always `true`

Reply:
That's only half the story though. If thats the argument, might as well remove `EnableSelfLogging` (since it isn't called right now) and everything associated with it.

Reply:
Revert this. As mentioned, this isn't unreachable in debug builds.

Reply:
If something is unused because it's only used by disabled debug code, don't remove it.

Also, as a baseline, please only open PRs with changes that you have reviewed yourself and deemed correct.

Reply:
I thought it was correct. What I meant by not knowing if it was correct was I don't know if is correct according to the project guidelines. Figuring out what issue is left intentionally is not easy

Reply:
LogLevel::LINFO is 4 and LogLevel::LDEBUG is 5, so it's currently unreachable even in debug builds.

I'm assuming this statement is guarding against the possibility that MAX_LOGLEVEL gets set to LWARNING, LERROR, or LNOTICE, which would make the INFO_LOG below pointless.

Reply:
Does that mean I should keep this change then?